lib/helpers.js

```diff
@@ -171,10 +171,17 @@ function getAuthFromSecretConfig(cfg, logger) {
   return returnConfig;
 }
 
+function removeSnapshotFromRequest(request) {
+  const cleanRequest = request;
```

Comment:
You probably need a deep clone for this to truly be a copy and not modify the original. I'm not sure if that was the intention or not

Reply:
Yep you're right. It was the intention. I was thinking that if the object was passed into a function, it was no longer a reference to its original self.